### PR TITLE
add custom job type so rake logs output

### PR DESCRIPTION
**Pull request by ajm188**

### CODE REVIEW

File: config/schedule.rb [hunk 1]
```diff
@@ -18,6 +18,8 @@
 # end
 
 # Learn more: http://github.com/javan/whenever
+job_type :log_rake, "cd :path && :environment_variable=:environment bundle exec rake :task :output > /proc/1/fd/1"
```
Comment (ajm188) [inline config/schedule.rb]: eh. this line probably won't be touched again

**Comment by matthewbentley** [comment]
So what I'm doing right now completely rips out whenever, so I'm not sure how useful this is

**Comment by ajm188** [comment]
Okay, well I'll just leave this as an alternative. We'll close it if we don't use it.

**Comment by matthewbentley** [comment]
Obsoleted by #186

**Pull request closed without merging**